src/sgame/sg_bot_nav.cpp

```diff
@@ -70,6 +70,8 @@ bool G_BotNavInit()
 		}
 	}
 	navMeshLoaded = true;
+	extern void BotAddSavedObstacles();
```

Comment:
This is hacky. Let's properly add things to header files if they are going to be called elsewhere.

Reply:
It's kind of a function that is only going to be called in one place, ever so that's why it's here. I guess I can move it before this function instead of inside it though.

Reply:
(done)